examples/usecases/usecase.tf

```diff
@@ -6,12 +6,12 @@
 terraform {
   required_providers {
     tanzu-mission-control = {
-      source = "vmware/tanzu-mission-control"
+      source = "vmware/dev/tanzu-mission-control"
     }
   }
 }
 
-// Create workspace
+# Create workspace
 resource "tanzu-mission-control_workspace" "create_workspace" {
```

Comment:
why is there a mixed style (snake and with `-` separator) naming for many resources?